pkg/projects/Microsoft.NETCore.UniversalWindowsPlatform/project.json

```diff
@@ -1,7 +1,7 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Portable.Compatibility": "1.0.2",
-    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.1-servicing-24904-01",
+    "Microsoft.NETCore.Runtime.CoreCLR": "1.1.2-servicing-25107-01",
```

Comment:
Do we want to update UWP meta-package? I guess we have till this point but it is something that needs closer review if we plan to ship it out of this branch in the future.